crates/dapf/README.md

```diff
@@ -0,0 +1,59 @@
+# dapf (DAP functions)
+
+This binary provides various functions that can be called to interact with DAP
+deployments.
+
+It several subcommands and their function can be consulted with the built in
+help pages the CLI provides.
+
+
+## Examples
+
+Here are some examples of common functionality of dapf. To avoid installing the
+tool replace all instances of `dapf` with `cargo run --bin dapf`.
+
+### Generate an hpke config
+
+With the default algorithm
+```
+dapf hpke generate
+```
+
+With a specific algorithm
+```
+dapf hpke generate x25519_hkdf_sha256
+```
+
+This will output the config twice
+```
+{
+  "config": {
+    "id": 131,
+    "kem_id": "x25519_hkdf_sha256",
+    "kdf_id": "hkdf_sha256",
+    "aead_id": "aes128_gcm",
+    "public_key": "3ccc3ada22dc0b3367d031ea4b02876a77a59ef32b5a1c06109552a51a739a13"
+  },
+  "private_key": "PRIVATE_KEY"
```

Comment:
nit: might as well leave the actual generated key in, since this is just a demonstration